BuildingSync.xsd

```diff
@@ -17787,4 +17790,468 @@
+                          <xs:simpleType>
+                            <xs:restriction base="xs:decimal">
+                              <xs:minInclusive value="0"/>
+                            </xs:restriction>
+                          </xs:simpleType>
+                        </xs:element>
+                        <xs:element name="CVRMSE" minOccurs="0" maxOccurs="1">
+                          <xs:annotation>
+                            <xs:documentation>The Coefficient of Variation of the Root Mean Square Error expressed as a percentage.</xs:documentation>
+                          </xs:annotation>
+                          <xs:simpleType>
+                            <xs:restriction base="xs:decimal">
+                              <xs:minInclusive value="0"/>
+                              <xs:maxInclusive value="100"/>
+                            </xs:restriction>
+                          </xs:simpleType>
+                        </xs:element>
+                        <xs:element name="NDBE" minOccurs="0" maxOccurs="1">
+                          <xs:annotation>
+                            <xs:documentation>The Net Determination Bias Error expressed as a percentage.</xs:documentation>
+                          </xs:annotation>
+                          <xs:simpleType>
+                            <xs:restriction base="xs:decimal">
+                              <xs:minInclusive value="0"/>
+                              <xs:maxInclusive value="100"/>
+                            </xs:restriction>
+                          </xs:simpleType>
+                        </xs:element>
+                        <xs:element name="MBE" type="xs:decimal" minOccurs="0" maxOccurs="1">
+                          <xs:annotation>
+                            <xs:documentation>The Mean Bias Error.</xs:documentation>
+                          </xs:annotation>
+                        </xs:element>
+                        <xs:element name="NMBE" minOccurs="0" maxOccurs="1">
+                          <xs:annotation>
+                            <xs:documentation>The Normalized Mean Bias Error expressed as a percentage.</xs:documentation>
+                          </xs:annotation>
+                          <xs:simpleType>
+                            <xs:restriction base="xs:decimal">
+                              <xs:minInclusive value="0"/>
+                              <xs:maxInclusive value="100"/>
+                            </xs:restriction>
+                          </xs:simpleType>
+                        </xs:element>
+                      </xs:sequence>
+                    </xs:complexType>
+                  </xs:element>
+                  <xs:element name="SummaryInformation" minOccurs="0">
+                    <xs:complexType>
+                      <xs:sequence>
+                        <xs:element name="NumberOfDataPoints" type="xs:decimal" minOccurs="0" maxOccurs="1">
```

Comment:
Datapoints should be one word these days... unless it is like DataPoint in buildingsync already.

Reply:
In G14 they use data points

Reply:
Leaving as `NumberOfDataPoints` to be consistent with G14.